docs/source/nodeoperations/restore.md

```diff
@@ -15,6 +15,15 @@ kubectl -n scylla-operator patch deployment/scylla-operator --type=json -p='[{"o
 
 :::
 
+:::{warning}
+Restoring schema with **ScyllaDB OS 5.4.X** or **ScyllaDB Enterprise 2024.1.X** and `consistent_cluster_management` isn’t supported.
+
+When creating the `target` ScyllaDB cluster, configure it with `consistent_cluster_management: false`.
+Refer to [API Reference](../api-reference/index.rst) to learn how to customize ScyllaDB configuration files.
```

Comment:
Worth mentioning that rolling restart is required, as there's no automatic one upon config change.

Reply:
It says "when creating the [...] cluster", and I put it before cluster creation step for that reason.

Reply:
ok, missed that. Thanks